openedx/features/student_certificates/views.py

```diff
@@ -194,8 +195,12 @@ def download_certificate_pdf(request, certificate_uuid):
 
     pdf_document_object = pdfkit.from_string(certificate_image_html, PDFKIT_IMAGE_PATH, pdfkit_options)
 
+    course_display_name = get_course_display_name_by_uuid(certificate_uuid)
+
     response_pdf_certificate = HttpResponse(pdf_document_object, content_type='application/pdf')
-    response_pdf_certificate['Content-Disposition'] = PDF_RESPONSE_HEADER.format(certificate_pdf_name='certificate')
+    response_pdf_certificate['Content-Disposition'] = PDF_RESPONSE_HEADER.format(
+        certificate_pdf_name='PhilanthropyUniversity_{course_name}'.format(
+            course_name=course_display_name.replace(' ', '')))
```

Comment:
Let's please simplify this piece of code to make it more readable. 
Make a new constant in constants for certificate pdf name 

`CERTIFICATE_PDF_NAME = 'PhilanthropyUniversity_{display_name}'`

and update these lines with

```
pdf_name = CERTIFICATE_PDF_NAME.format(display_name=course_display_name.replace(' ', ''))
response_pdf_certificate['Content-Disposition'] = PDF_RESPONSE_HEADER.format(certificate_pdf_name=pdf_name)
```